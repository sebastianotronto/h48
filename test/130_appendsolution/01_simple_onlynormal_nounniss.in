U F R D2 B'
0
1
rotation UF
0
