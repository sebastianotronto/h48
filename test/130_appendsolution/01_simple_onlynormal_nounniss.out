U F R D2 B'
Number of solutions: 1
Shortest solution length: 5
Used bytes: 12
